src/main/java/org/ihtsdo/buildcloud/core/service/build/Rf2FileExportRunner.java

```diff
@@ -216,6 +217,26 @@ private void generateReleaseFile(final String transformedDeltaDataFile, final Ma
 				LOGGER.debug("Finish: Insert previous release data into table {}", tableSchema.getTableName());
 			}
 
+			if (!StringUtils.isEmpty(configuration.getAdditionalPreviousPublishedPackages())) {
```

Comment:
@QuyenLy87 Do we need to include all files from additional previous packages or just for input files exist in additional previous package?